.circleci/config.yml

```diff
@@ -1,11 +1,29 @@
 version: "2.1"
 
 commands:
+  common_setup:
+    description: Prepare the repository for work
+    steps:
+      - checkout
+      - run: sudo pip install pipenv
+      - run: make init
+
   run_tests:
     description: Run test suite
     steps:
       - common_setup
       - run: make unit_test
+      - run: make integration_test
+
+  run_integration_tests_using_released_opera_version:
+    description: Test xOpera CLI commands with testing service template
+    steps:
+      - checkout
+      - run: pip install --user --upgrade pip
+      - run: pip install --user wheel
+      - run: pip install --user opera
+      - run: export OPERA=opera
+      - run: make integration_test
```

Comment:
Are you sure this works? I never looked into the CircleCI internals, but I would guess that each run command is executed in a separate subprocess, making it impossible to define a variable in one command and use it in the next one. It would be safer just to do `run: OPERA=opera make integration_test` and avoid any pitfalls of the CI environment here.

Reply:
Well well...I'm not so sure. I remember that with every `run` on a separate line a new shell is opened so I agree that we can join this two lines to avoid any possible trouble.